qiskit_nature/operators/__init__.py

```diff
@@ -11,31 +11,16 @@
 # that they have been altered from the originals.
 
 """
-Operators for Qiskit Nature (:mod:`qiskit_nature.operators`)
-============================================================
+Operators (:mod:`qiskit_nature.operators`)
+==========================================
 
 .. currentmodule:: qiskit_nature.operators
 
-Operators
-==========
-
-Second-Quantization Operators
-+++++++++++++++++++++++++++++
+Operators for different systems such as fermionic, vibrational and spin.
```

Comment:
Especially here I find this list a little bit confusing. If you do want to mention examples here, you should also make clear how they differ. But in fact I think it is not needed here at all.

Reply:
I think we should have some text in these sections that gives a bit of an overview; not everyone that looks over the API ref is as knowledgeable on the topic. I put a single sentence just to describe what this has and I agree things could be improved. Maybe differences are not needed since they should say individually in the class what they are for etc and differences would be seen. But some aspects might be useful like, since fermions have spin why is that not covered by the spin op that it is special case, as in things maybe not so clear to some people.